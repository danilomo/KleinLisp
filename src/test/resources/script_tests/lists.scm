(define lst '(1 2 3 4))
(println (car lst))
(println (cdr lst))
(println (cons 0 lst))
(println (length lst))
(println (null? '()))
(println (null? lst))