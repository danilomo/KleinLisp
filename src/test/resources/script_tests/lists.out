1
(2 3 4)
(0 1 2 3 4)
4
true
false